fastmri_recon/data/utils/multicoil/smap_extract.py

```diff
@@ -45,3 +45,15 @@ def extract_smaps(kspace, low_freq_percentage=8, background_thresh=4e-6):
     # for now we do not perform background removal based on low_freq_rss
     # could be done with 1D k-means or fixed background_thresh, with tf.where
     return coil_smap
+
+
+def non_cartesian_extract_smaps(kspace, trajs, dcomp, nufft_back, low_freq_percentage=8):
+    cutoff_freq = low_freq_percentage / 100 * tf.constant(pi)
```

Comment:
Maybe we should divide by 2 here since the full kspace is 2 pi. What do you think?

Reply:
Its a matter of convention, if you are following this, maybe I will keep it the same for uniformity